maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -510,7 +503,8 @@ void logging(CliRequest cliRequest) {
         cliRequest.showErrors = cliRequest.verbose || commandLine.hasOption(CLIManager.ERRORS);
 
         // LOG COLOR
-        String styleColor = cliRequest.getUserProperties().getProperty(STYLE_COLOR_PROPERTY, "auto");
+        String styleColor = cliRequest.getUserProperties().getProperty("style.color", "auto");
+        styleColor = cliRequest.getUserProperties().getProperty(Constants.MAVEN_STYLE_COLOR_PROPERTY, styleColor);
```

Comment:
Is that on purpose?

Reply:
Yes, I wanted all public properties in `Constants` to start with `maven.`.  That one was an exception, so it's now used if the new correct one isn't set.